### ISSUE TITLE
Removing a field from "fullTextSearchFields" does not prevent indexing

### ISSUE THREAD

**Issue by wyattis** [nuxt/content#925]
### Version
@nuxt/content: 1.14.0
nuxt: 2.15.7

### Reproduction Link
https://codesandbox.io/s/nostalgic-snow-54by8?file=/nuxt.config.js

### Steps to reproduce
1. Add "fullTextSearchFields" to nuxt.config.js without "description".
```
content: {
  fullTextSearchFields: ['title', 'slug', 'text']
}
```
2. Add some content in the content directory with an object for description instead of text.
```
---
title: Test title
description: { body: A *markdown* description }
---
Lorem ipsum
```
3. `npm run dev`

### What is Expected?
The file should be processed without an error because the description should be skipped.

### What is actually happening?
There is an error: 
```
✖ Nuxt Fatal Error      

TypeError: Unsupported field type for full text search.
```

**Comment by NozomuIkuta** [comment]
TL;DR

This is not a bug but a spec.
See [the related part in the official documentation](https://content.nuxtjs.org/configuration#merging-defaults).

---

If you want to know details:

Nuxt Content uses `arrayFn()` method of [`defu` package](https://github.com/unjs/defu#readme) to merge user options and default ones.

By default, `defu.arrayFn()` method concatenates arrays instead of replacing one with another.

If you want to override default array completely or modify it arbitrarily, you can define a function that receives the default array and return an array (modified or new one).

```javascript
export default {
  content: {
    fullTextSearchFields: ['title', 'slug', 'text']
  }
}

// will yields

{
  content: {
    fullTextSearchFields: [
      'title',
      'description',
      'slug',
      'text',
      'title',
      'slug',
      'text'
    ]
  }
}
```

```javascript
export default {
  content: {
    fullTextSearchFields: () => ['title', 'slug', 'text']
  }
}

// will yields

{
  content: {
    fullTextSearchFields: [
      'title',
      'slug',
      'text'
    ]
  }
}
```

**Comment by wyattis** [comment]
I missed that part of the docs. Thanks!
